cmd/baseserver/Makefile

```diff
@@ -0,0 +1,15 @@
+all: install linux darwin windows
```

Comment:
Do we need those CGO_ENABLED flags ?

Also, these all look like the same command. Do we need them to be separate or can we just have one make command?